ui-lib.pl

```diff
@@ -1442,7 +1442,7 @@ =head2 ui_buttons_start
 sub ui_buttons_start
 {
 return &theme_ui_buttons_start(@_) if (defined(&theme_ui_buttons_start));
-return "<table width='100%' class='ui_buttons_table'>\n";
+return "<table width='100%' class='ui_buttons_table'>\n<tr><td>";
```

Comment:
Is this compatible with the HTML generated by ui_buttons_row , which starts with <tr> ?